airbyte-integrations/connectors/source-mysql/mysql-script.sql

```diff
@@ -0,0 +1,251 @@
+set @lastinsertcounter = 0;
+set @lastinsert = 0;
+set @fullloop = 0;
+set @fullloopcounter = 0;
+
+while @vmaxx < @vmaxoneinsert do
+	set @vmaxoneinsert = @vmaxx;
+	set @fullloop = @fullloop + 1;
+end while;
+commit;
+
+while @vmax > @vmaxoneinsert do
+	set @fullloop = @fullloop + 1;
+	set @vmax = @vmax - @vmaxoneinsert;
+	set @lastinsert = @vmax;
+end while;
+commit;
+
+set @insertTable = concat('insert into test (varchar1, varchar2, varchar3, varchar4, varchar5, longblobfield, timestampfield) values (');
+while @counter < @vmaxoneinsert do
+	set @insertTable = concat(@insertTable, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP), (');
+	set @counter = @counter + 1;
+end while;
+commit;
+set @insertTable = concat(@insertTable, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP);');
+
+while @vmax < 1 do
+	set @fullloop = 0;
+	set @vmax = 1;
+end while;
+commit;
+
+while @fullloopcounter < @fullloop do
+	PREPARE runinsert from @insertTable;
+	EXECUTE runinsert;
+	DEALLOCATE PREPARE runinsert;
+	set @fullloopcounter = @fullloopcounter + 1;
+end while;
+commit;
+
+set @insertTableLasted = concat('insert into test (varchar1, varchar2, varchar3, varchar4, varchar5, longblobfield, timestampfield) values (');
+while @lastinsertcounter < @lastinsert do
+	set @insertTableLasted = concat(@insertTableLasted, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP), (');
+	set @lastinsertcounter = @lastinsertcounter + 1;
+end while;
+commit;
+set @insertTableLasted = concat(@insertTableLasted, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP);');
+
+while @lastinsert > 0 do
+	PREPARE runinsert from @insertTableLasted;
+	EXECUTE runinsert;
+	DEALLOCATE PREPARE runinsert;
+	set @lastinsert = 0;
+end while;
+commit;
+
+end #
+
+delimiter ;
+
+delimiter #
+create procedure table_create()
+begin
+
+create table test
+(
+id int unsigned not null auto_increment primary key,
+varchar1 varchar(255),
+varchar2 varchar(255),
+varchar3 varchar(255),
+varchar4 varchar(255),
+varchar5 varchar(255),
+longblobfield longblob,
+timestampfield timestamp
+)
+engine=innodb;
+
+set @extraSmallText = '\'test weight 50b - 50b text, 50b text, 50b text\'';
+set @smallText = '\'test weight 500b - 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text\'';
```

Comment:
For MySQL, you can use the [REPEAT](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_repeat) function to generate these repetitive strings.

Reply:
updated it